config/repo-values.yaml

```diff
@@ -502,3 +502,5 @@ sync:
       url: https://charts.streamnative.io/
     - name: kured
       url: https://weaveworks.github.io/kured
+    - name: unofficial-superset-snowflake
```

Comment:
The name can only be 21 characters long. This one is 29. Can you please use a shorter name.

Reply:
Cross-referencing https://github.com/helm/hub/issues/362